scout/parse/variant/transcript.py

```diff
@@ -229,13 +229,19 @@ def parse_transcripts(raw_transcripts, allele=None):
         transcript["dbsnp"] = []
         transcript["cosmic"] = []
         variant_ids = entry.get("EXISTING_VARIATION")
+
         if variant_ids:
             for variant_id in variant_ids.split("&"):
                 if variant_id.startswith("rs"):
                     transcript["dbsnp"].append(variant_id)
                 elif variant_id.startswith("COSM") or variant_id.startswith("COSV"):
                     transcript["cosmic"].append(variant_id)
 
+        cosmic_ids = entry.get("COSMIC")
+        if cosmic_ids:
+            for cosmic_id in cosmic_ids.split("&"):
+                transcript["cosmic"].append(cosmic_id)
```

Comment:
Sometimes there are duplicate entries in this field. Would it create problems? Example: 
![image](https://user-images.githubusercontent.com/1086877/131969263-40fe1bdb-18f4-4edd-9bce-937e0ddc3bd8.png)

Reply:
I'll Slack you the link for this particular case

Reply:
good point, I'll fix!

Reply:
Checked and this is already covered by the code, for instance this variant that has repeated cosmic ID (COSV51973290&COSV51973290&COSV51973290)

`17	16075254	.	C	T	271	PASS	SAMPLE=aliveswift;TYPE=SNV;DP=1790;VD=175;AF=0.0978;BIAS=1:2;REFBIAS=59:1555;VARBIAS=12:163;PMEAN=12;PSTD=1;QUAL=36.5;QSTD=1;SBF=0.06216;ODDRATIO=1.93945;MQ=60;SN=350;HIAF=0.0981;ADJAF=0.0011;SHIFT3=0;MSI=2;MSILEN=1;NM=2;HICNT=175;HICOV=1783;LSEQ=CGATTCCAGTGAATCATGAT;RSEQ=CACTGGGGATGGGCCTGGAT;DUPRATE=0;SPLITREAD=0;SPANPAIR=0;CSQ=T|missense_variant|MODERATE|NCOR1|ENSG00000141027|Transcript|ENST00000268712|protein_coding|4/46||ENST00000268712.3:c.298G>A|ENSP00000268712.2:p.Asp100Asn|556|298|100|D/N|Gat/Aat|COSM4541346&COSM4541347||-1||SNV|HGNC|7672|YES|||CCDS11175.1|ENSP00000268712|O75376|Q9NSZ0&C9JAP0&B4DJ25|UPI000013D7D5||Ensembl|C|C||1|deleterious(0)|possibly_damaging(0.884)|hmmpanther:PTHR13992&hmmpanther:PTHR13992:SF5|||||||||||||||||||||||1&1|1&1||||||COSV51973290&COSV51973290&COSV51973290|c.298G>A&c.109-6779G>A&c.298G>A|NCOR1_ENST00000395851&NCOR1_ENST00000395848&NCOR1|-&-&-|1&1&1|p.D100N&p.?&p.D100N,T|missense_variant|MODERATE|NCOR1|9611|Transcript|XM_005256867.1|protein_coding|4/48||XM_005256867.1:c.298G>A|XP_005256924.1:p.Asp100Asn|387|298|100|D/N|Gat/Aat|COSM4541346&COSM4541347||-1||SNV|EntrezGene|7672|YES||||XP_005256924.1|||||RefSeq|C|C||||||||||||||||||||||||||||1&1|1&1||||||COSV51973290&COSV51973290&COSV51973290|c.298G>A&c.109-6779G>A&c.298G>A|NCOR1_ENST00000395851&NCOR1_ENST00000395848&NCOR1|-&-&-|1&1&1|p.D100N&p.?&p.D100N	GT:DP:VD:AD:AF:RD:ALD	0/1:1790:175:1614,175:0.0978:59,1555:12,163`

with the parsing of this branch will look like this:
![image](https://user-images.githubusercontent.com/28093618/131975098-91e6b827-b015-4915-bbdb-f819c8a2bde1.png)

That's because the list of cosmic IDs is converted into a set when the transcripts are returned to the main variant parsing code: https://github.com/Clinical-Genomics/scout/blob/a8db52daf09696fc77087917695fe5e2e522985b/scout/parse/variant/variant.py#L278

Reply:
So there is no issue here with duplicate COSMIC IDs. 👍🏽

Reply:
Nope!